app/javascript/src/NewService/types/index.js

```diff
@@ -9,3 +9,8 @@ export type FormProps = {
   submitText: string,
   children?: React.Node
 }
+
+export type BackendApi = {
```

Comment:
Use type `Api` from `Types` instead (perhaps you'll need to turn `link` and `type` into optionals)

Reply:
@josemigallas means from `app/javascript/src/Types/Api.jsx`